code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -167,11 +167,13 @@
 		..()
 
 var/list/blacklisted_pylon_turfs = typecacheof(list(
-	/turf/simulated/floor/engine/cult,
-	/turf/space,
-	/turf/simulated/floor/plating/lava,
-	/turf/simulated/floor/chasm,
-	/turf/simulated/wall,
+    /turf/simulated/floor/engine/cult,
+    /turf/space,
+    /turf/simulated/floor/plating/lava,
+    /turf/simulated/floor/chasm,
+    /turf/simulated/wall/cult,
+    /turf/simulated/wall/cult/artificer,
+    /turf/unsimulated/wall
```

Comment:
Dunno what happened here - I doubt it causes any issues, though, seeing as I was able to test it successfully.